### ISSUE TITLE
automate inclusion in TeSS

### ISSUE THREAD

**Issue by ac812** [elixir-europe-training/ELIXIR-TrP-LessonTemplate-MkDocs#19]
Talk to the TeSS team to see if given that the training lesson has a Bioschema in it, is there a way to automate the inclusion in TeSS?
Message from Patricia:  Currently Glittr is in discussion with the TeSS Team to see if the training materials in Glitter can be automatically added to TeSS.

**Comment by GeertvanGeest** [comment]
That's correct regarding the glittr remark. If there will be direct bioschemas support from the side of glittr, the bioschema record will be based on what it can find at the github/lab api (i.e. what's written in a.o. the 'description' and 'title' in the gh repository) + the topics that are assigned by glittr -> we would basically reformat https://glittr.org/api/repositories. It will not parse any schema records in the repo/website itself. However, glittr could be used to point to repositories/websites that then can be parsed by TeSS ..

**Comment by GeertvanGeest** [comment]
If we standardize the use of bioschemas in the templates, there is no 'easy' way (I think?) for TeSS to scrape them all, including new ones. We can automate collecting bioschemas markup from this GitHub organization and emit them at elixir-europe-training.github.io.
